<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>769</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="Bluetooth">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>251</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Futura</family>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Port Choice</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QComboBox" name="bluetooth_box">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Futura</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="bluetooth_confirm">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Futura</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="View">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>251</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Futura</family>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Gain</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="R2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Futura</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="R1">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Futura</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="send">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>170</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Futura</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>130</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="minimum">
      <number>3</number>
     </property>
     <property name="maximum">
      <number>801</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Value">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>150</y>
      <width>511</width>
      <height>491</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Futura</family>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Mesures</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="value">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Futura</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="gain">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Futura</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Run">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>150</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Futura</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Run</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Save">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>180</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Futura</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QWidget" name="graphWidget" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>201</width>
       <height>151</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="graph_container" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>331</width>
       <height>301</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="_quit">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>670</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Futura</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="Arduino_sensor">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>321</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Futura</family>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Port Selected</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="arduino_name">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Futura</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>37</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
